Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -15,69 +15,73 @@
 using Windows.Security.ExchangeActiveSyncProvisioning;
 using Windows.System;
 using Windows.System.Profile;
+using Windows.System.UserProfile;
 
 namespace Microsoft.Toolkit.Uwp.Helpers
 {
     /// <summary>
     /// Defines class providing information of OS and application
     /// </summary>
-    public class SystemInformation
+    public static class SystemInformation
     {
         /// <summary>
         /// Gets Application's name
         /// </summary>
-        public string ApplicationName { get; private set; }
+        public static string ApplicationName { get; }
 
         /// <summary>
         /// Gets Application's version
         /// </summary>
-        public PackageVersion ApplicationVersion { get; private set; }
+        public static PackageVersion ApplicationVersion { get;  }
 
         /// <summary>
         /// Gets current culture
         /// </summary>
-        public string Culture { get; private set; }
+        public static string Culture { get; }
```

Comment:
It will be better, though I will still convert this from [GlobalizationPreferences.Languages](https://msdn.microsoft.com/pl-pl/library/windows/apps/windows.system.userprofile.globalizationpreferences.languages) as it returns the most preferred language by the user, rather than _System.Globalization.CultureInfo.CurrentCulture_ - [some more info](http://stackoverflow.com/a/15282352/2681948).

I've also changed the summary of the property and I'm pushing the change.